script/check-style

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+#
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+#
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+
+CLANG_FORMAT="$(which clang-format)" || CLANG_FORMAT="${HOME}/clang/bin/clang-format"
+
+CLANG_VERSION="$(${CLANG_FORMAT} -version | cut -d ' ' -f 3)"
+CLANG_VERSION_REQUIRED="3.8.0"
+if [[ "${CLANG_VERSION}" != "${CLANG_VERSION_REQUIRED}" ]]; then
+  echo "Current version: ${CLANG_VERSION}"
+  CLANG_DIRECTORY=$(mktemp -d /tmp/clang_XXXXX)
+  echo "Installing required clang-format ${CLANG_VERSION_REQUIRED} to ${CLANG_DIRECTORY}"
+  curl --silent --show-error --retry 10 \
+    "http://releases.llvm.org/${CLANG_VERSION_REQUIRED}/clang+llvm-${CLANG_VERSION_REQUIRED}-x86_64-linux-gnu-debian8.tar.xz" \
```

Comment:
our Travis uses Ubuntu Xenial (16.04), so this should be: http://releases.llvm.org/3.8.0/clang+llvm-3.8.0-x86_64-linux-gnu-ubuntu-16.04.tar.xz